build.sbt

```diff
@@ -32,7 +32,9 @@ lazy val scalaPyTensorFlowCross = crossProject(JVMPlatform, NativePlatform)
   .in(file("tensorflow"))
   .settings(
     name := "scalapy-tensorflow-cross",
-    libraryDependencies += "me.shadaj" %%% "scalapy-numpy" % "0.1.0+6-14ca0424",
+    // scalapy-core version should be consistent with what our version of scalapy-numpy uses
```

Comment:
This comment isn't true, quite the opposite - we're explicitly choosing a different version of `scalapy-core` than what would be transitively pulled in by `scalapy-numpy`